es_source.go

```diff
@@ -133,13 +152,44 @@ func boundsFilter(geometryField string, tile maptile.Tile) *Dict {
 	}
 }
 
+// Given the list of extra source arguments that were specified with request, transform
+// these into a map of property name to ES document source path, or return an error
+// if there is a malformed extra source argument.
+func makeFieldMap(incArgs []string) (map[string]string, error) {
+	var result = make(map[string]string);
+
+	for _, source := range incArgs {
+		splits := strings.SplitN(source, ":", 2);
```

Comment:
This syntax definitely has the benefit of resembling both ES-style query parameter values and our YAML configuration syntax :+1:

Reply:
Thanks!